server/chat/background/task.py

```
@@ -1257,39 +1328,33 @@ async def _execute_background_chat(
-            weaviate_client=weaviate_client,
-            postgres_client=postgres_client,
-            websocket_sender=None,
-            event_loop=None,
-            ctx_len=15,  # Reasonable history for RCAs - allows agent to see investigation progress
-        )
-        logger.info(f"[BackgroundChat] Created agent with ctx_len=15 (no WebSocket)")
-        
         # Create workflow for this session
         wf = Workflow(agent, session_id)
-        logger.info(f"[BackgroundChat] Created workflow for session {session_id}")
+        logger.info(f"[BackgroundChat] Created agent + workflow (singleton reuse)")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)